src/client/Microsoft.Identity.Client/Platforms/Features/WebView2WebUi/WinFormsPanelWithWebView2.cs

```diff
@@ -214,10 +214,12 @@ private bool CheckForEndUrl(Uri url)
             if (!readyToClose &&
                 !url.Scheme.Equals("https", StringComparison.OrdinalIgnoreCase) &&
                 !url.AbsoluteUri.Equals("about:blank", StringComparison.OrdinalIgnoreCase) &&
-                !url.Scheme.Equals("javascript", StringComparison.OrdinalIgnoreCase))
+                !url.Scheme.Equals("javascript", StringComparison.OrdinalIgnoreCase) &&
+                !url.Scheme.Equals("res", StringComparison.OrdinalIgnoreCase) // IE error pages
+                )
             {
 
-                _logger.Error($"Redirection to non-HTTPS scheme ({url.Scheme}) found! WebView will fail...");
+                _logger.Error($"Redirection to non-HTTPS scheme ({url}) found! WebView2 will fail...");
```

Comment:
nit: maybe add "[WebView2]" log prefix here (or if there's already one that we use.
Also maybe reword a bit - change 'scheme' to 'url`, since that's what's being printed? URL is non-PII too, right?